buildSrc/subprojects/plugins/src/main/kotlin/org/gradle/plugins/buildtypes/BuildTypesPlugin.kt

```diff
@@ -110,7 +110,7 @@ fun Project.insertBuildTypeTasksInto(
     }
 
     if (taskList.isEmpty()) {
-        taskList.add("help") //do not trigger the default tasks
+        taskList.add("help") // do not trigger the default tasks
```

Comment:
:x: It would be nicer if the changes in spaces were committed separately (no need to go through a PR for those).